src/test/fuzz/orphanage.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <net_processing.h>
+#include <primitives/transaction.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/mining.h>
+#include <test/util/script.h>
+#include <test/util/setup_common.h>
+#include <txorphanage.h>
+#include <validation.h>
+
+
+FUZZ_TARGET(orphanage)
+{
+    FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
+
+    TxOrphanage orphanage;
+    std::set<uint256> orphan_work_set;
+    std::set<COutPoint> outpoints;
```

Comment:
I think this might be better as std::vector, to get a more stable, predictable sorting

Reply:
`vector` is better if we only want to add outpoints to the end but it will be slow if we want to randomly delete elements.